src/lib/stepper/stepper-button.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Directive} from '@angular/core';
+import {CdkStepperNext, CdkStepperPrevious} from '@angular/cdk';
+import {MdStepper} from './stepper';
+
+@Directive({
+  selector: 'button[mdStepperNext], button[matStepperNext]',
+  host: {
+    '(click)': '_onClick()',
+  }
+})
+export class MdStepperNext extends CdkStepperNext {
```

Comment:
Does it work if you just do this?
```ts
@Directive({selector: 'button[mdStepperNext], button[matStepperNext]'})
export class  MdStepperNext extends CdkStepperNext { }
```
I thought the click handler would be inherited automatically and that the constructor wouldn't be necessary if `MdStepper` provides itself as `CdkStepper`